### PR TITLE
slycot 0.3.3

### PR DESCRIPTION
**Pull request by murrayrm**
Checklist
* Used a fork of the feedstock to propose changes
* Reset the build number to `0` (if the version changed)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,26 +13,30 @@ source:
...
 
 requirements:
   build:
-    - {{ compiler('fortran') }}
+    - {{ compiler('fortran') }}  # [not win]
     - {{ compiler('c') }}
+    - flang  # [win]
+    # on Windows, this relies on having visual studio CE 2015
+    # this link needed quite some searching, please do not delete!
+    # https://go.microsoft.com/fwlink/?LinkId=532606&clcid=0x409
```
Comment (moorepants) [inline recipe/meta.yaml]: I think that `{{ compiler('fortran') }}` should load `flang`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -13,26 +13,30 @@ source:
...
+
   host:
     - python
-    - blas 1.1 {{ variant }}
-    - openblas
     - numpy
+    - libflang  # [win]
+    - libgfortran  # [not win]
+    - lapack
```
Comment (moorepants) [inline recipe/meta.yaml]: These change is incorrect I believe. The `compiler()` flags should handle correctly.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {% set name = "slycot" %}
-{% set version = "0.3.0" %}
-{% set sha256 = "6be508f5ac1159da9011de7764df0f82bffdda6ca0899182ec12d039b81c831a" %}
-{% set variant = "openblas" %}
+{% set version = "0.3.3" %}
+{% set sha256 = "d98ec859a398af8a3917a0d62e0724996c8b3e780813f8685410525aed9fbafd" %}
+{% set variant = "lapack" %}
```
Comment (moorepants) [inline recipe/meta.yaml]: I think this should stay as `openblas`.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -13,26 +13,30 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 202
-  # NOTE: Conda forge lacks openblas on Windows, so skip the Windows build.
-  skip: true  # [win]
-  features:
-    - blas_{{ variant }}
```
Comment (isuruf) [inline recipe/meta.yaml]: Keep this feature. If you remove this, older builds will be given priority if `blas=*=openblas` package is installed.

File: recipe/meta.yaml [hunk 5]
```diff
@@ -13,26 +13,30 @@ source:
...
+    - {{ compiler('fortran') }}  # [not win]
     - {{ compiler('c') }}
+    - flang  # [win]
+    # on Windows, this relies on having visual studio CE 2015
+    # this link needed quite some searching, please do not delete!
+    # https://go.microsoft.com/fwlink/?LinkId=532606&clcid=0x409
+
   host:
     - python
-    - blas 1.1 {{ variant }}
```
Comment (isuruf) [inline recipe/meta.yaml]: Don't remove this.

File: recipe/meta.yaml (hunk 2 above)
Comment (isuruf) [reply recipe/meta.yaml]: Yes, `libflang` and `libgfortran` can be removed. `lapack` should be replaced with `openblas`

File: recipe/meta.yaml [hunk 6]
```diff
@@ -13,26 +13,30 @@ source:
...
+    - lapack
+
   run:
-    - python
-    - blas 1.1 {{ variant }}
-    - openblas
     - {{ pin_compatible('numpy') }}
+    - lapack
+    - libgfortran  # [not win]
+    - libflang  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]: Remove 2 lines above. Change lapack->openblas and keep `blas 1.1 {{variant}}` line

**Comment by murrayrm** [comment]
@moorepants Thanks for taking a look at this.  Updated to reflect comments.  Still waiting on the MacOS build (not sure why that is hanging; perhaps just a long queue of jobs ahead of it?).

**Pull request merged**